src/frameworks.js

```diff
@@ -23,11 +23,13 @@ module.exports = [
     name: 'Nuxt.js V3 (Vue.js V3)',
     value: 'nuxtjs-3',
     start: 'dev',
-    token: 'd6IKUtAUDiKyAhpJtrLFcwtt',
+    token: 'W1vLyxT5rQ15jBpANjnv0gtt',
     config: 'nuxt.config.js',
-    bridge: 'pages/Home.vue',
+    bridge: 'pages/index.vue',
     public: 'public',
     port: '3000',
+    https: true,
```

Comment:
This probably needs to be `http` or the `npm run dev` command needs to start `https` automatically

Reply:
Okay, set to `false` now. There's a `npm run dev-ssl` command included in the Nuxt 3 repository, so I've updated the start command to that.